smoke-tests/wildfly/src/test/groovy/io/opentelemetry/smoketest/WildflySmokeTest.groovy

```diff
@@ -18,8 +18,13 @@ class WildflySmokeTest extends AbstractServerSmokeTest {
     ProcessBuilder processBuilder =
       new ProcessBuilder("${wildflyDirectory}/bin/standalone.sh")
     processBuilder.directory(wildflyDirectory)
+
+    // We're installing a span exporter to make sure it doesn't blow anything up, but we're not
+    // checking the spans, since JBoss seems to redirect stdout to something we don't have (easy) access to.
```

Comment:
someday we'll pipe the spans over-the-wire to a listener (instead of piping them via-log-file). until then, this is good

Reply:
Agreed.